gamemodes/terrortown/entities/weapons/weapon_ttt_c4.lua

```diff
@@ -22,6 +22,37 @@ if CLIENT then
 
     SWEP.Icon = "vgui/ttt/icon_c4"
     SWEP.IconLetter = "I"
+else
+    ---
+    -- @realm server
+    local c4_radius = CreateConVar(
+        "ttt2_c4_radius",
+        "600",
+        { FCVAR_NOTIFY, FCVAR_ARCHIVE },
+        "Defines the damage radius of C4 explosions"
+    )
+
+    ---
+    -- @realm server
+    local c4_inner_radius = CreateConVar(
+        "ttt2_c4_radius_inner",
+        "500",
+        { FCVAR_NOTIFY, FCVAR_ARCHIVE },
+        "Defines the kill radius of C4 explosions"
+    )
+
+    hook.Add("TTT2SyncGlobals", "TTT2SyncC4Globals", function()
+        SetGlobalInt(c4_radius:GetName(), c4_radius:GetInt())
+        SetGlobalInt(c4_inner_radius:GetName(), c4_inner_radius:GetInt())
+    end)
+
+    cvars.AddChangeCallback(c4_radius:GetName(), function(name, old, new)
+        SetGlobalInt(c4_radius:GetName(), tonumber(new))
+    end, c4_radius:GetName())
+
+    cvars.AddChangeCallback(c4_inner_radius:GetName(), function(name, old, new)
+        SetGlobalInt(c4_inner_radius:GetName(), tonumber(new))
+    end, c4_inner_radius:GetName())
```

Comment:
you don't need that, you can remove that part. If you need a shared convar, use replicated cvars. Global vars were a workaround years ago when replication was broken